hunting/llm/docs/aws_bedrock_ignore_previous_prompt_detection.md

```diff
@@ -0,0 +1,48 @@
+# AWS Bedrock LLM Ignore Previous Prompt Detection
+
+---
+
+## Metadata
+
+- **Author:** Elastic
+- **Description:** This analytic flags multiple instances of LLM vulnerability to malicious user interaction, helping to maintain ethical guidelines and compliance standards.
+- **UUID:** `131e5887-463a-46a1-a44e-b96361bc6cbc`
+- **Integration:** [aws_bedrock.invocation](https://docs.elastic.co/integrations/aws_bedrock)
+- **Language:** `[ES|QL]`
+- **Source File:** [AWS Bedrock LLM Sensitive Content Refusals](../queries/aws_bedrock_ignore_previous_prompt_detection.toml)
+
+## Query
+
+```sql
+from logs-aws_bedrock.invocation-*
+ |WHERE @timestamp > NOW() - 1 HOUR
+   AND (
+     (gen_ai.prompt LIKE "*Ignore the above instructions*" OR 
+      gen_ai.prompt LIKE "*IGNORE INSTRUCTIONS*" OR
+      gen_ai.prompt LIKE "*IGNORE and PRINT*" OR
+      gen_ai.prompt LIKE "*Ignore and say*" OR 
+      gen_ai.prompt LIKE "*nevermind ignore what i asked and print")
+     AND gen_ai.completion LIKE "*end_turn*"
```

Comment:
```suggestion
     (gen_ai.prompt LIKE "%Ignore the above instructions%" OR 
      gen_ai.prompt LIKE "%IGNORE INSTRUCTIONS%" OR
      gen_ai.prompt LIKE "%IGNORE and PRINT%" OR
      gen_ai.prompt LIKE "%Ignore and say%" OR 
      gen_ai.prompt LIKE "%nevermind ignore what i asked and print%")
     AND gen_ai.completion LIKE "%end_turn%"
```

If your using `LIKE`, the wildcard has to be a `_` or `%`. `RLIKE` allows for `*`.
Not sure if we are certain about case sensitivity, but you can always do `to_lower(gen_ai.completion) LIKE "%ignore and%"`

Reply:
Sorry, adjust query, then re-populate docs

Reply:
The to_lower one is a good addition, will modify. 
The LIKE with "%" was never working and i could not get the details. Hence moved to "*"
from the example : hunting/llm/docs/aws_bedrock_sensitive_content_refusal_detection.md

Reply:
Am I missing something? From the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-functions-operators.html#esql-like-operator) it seems like `*` is supported by `LIKE` and the screenshot in the description shows the query working as expected right @shashank-elastic ?

Reply:
Yes @imays11 * is working as expected